packages/core/http/core-http-router-server-internal/src/versioned_router/core_versioned_route.ts

```diff
@@ -117,17 +114,14 @@ export class CoreVersionedRoute implements VersionedRoute {
     this.enableQueryVersion = options.enableQueryVersion === true;
     this.defaultSecurityConfig = validRouteSecurity(options.security, options.options);
     this.options = options;
-    this.router[this.method](
-      {
-        path: this.path,
-        validate: passThroughValidation,
-        // @ts-expect-error upgrade typescript v5.1.6
-        options: this.getRouteConfigOptions(),
-        security: this.getSecurity,
-      },
-      this.requestHandler,
-      { isVersioned: true, events: false }
```

Comment:
Another change: removal of opaque option `events` that created conditional flows behind indirection